### PR TITLE
php72-http: Add pecl_http 3.1.0 (new formula)

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/php72-http.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  head "https://github.com/m6w6/ext-http.git"
+
+  bottle do
+    sha256 "0fcb8213d5d934f5a1c41fdb1a492d0cd72a0a52213e0c0f8ffb5c362b5b85e6" => :sierra
+    sha256 "87c97ac475bc33b4208aff16dab4dd287379f13b5a4fdde63760f78bbd8a6ad9" => :el_capitan
+    sha256 "fa96d6f0bfc845d946747bb4c705e1c2a818719405d086610adc77a266bcb45b" => :yosemite
+  end
+
+  depends_on "libevent"
+  depends_on "php70-intl"
```
Comment (javian) [inline Formula/php72-http.rb]: You need to depend on the php72 versions of these formulas for it to work

**Comment by SMillerDev** [comment]
@javian fixed

**Comment by SMillerDev** [comment]
depends on #4268 now

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
There seems to be some issue with propro and raph extension - can you include them as revision bumps as seperate commits in this PR ?

**Comment by SMillerDev** [comment]
@javian done

**Comment by SMillerDev** [comment]
this will now conflict for #4380 though

**Comment by javian** [comment]
@smillerdev yes but we can exclude these formulas from that since it needs to be reorganized anyway.

**Comment by javian** [comment]
Thanks @smillerdev !

**Pull request closed without merging**